x-pack/test/security_solution_api_integration/test_suites/detections_response/basic_essentials_license/alerts/index.ts

```diff
@@ -7,11 +7,9 @@
 import { FtrProviderContext } from '../../../../ftr_provider_context';
 
 export default function ({ loadTestFile }: FtrProviderContext) {
-  describe('Detection Engine Basic and Essentials API', function () {
-    loadTestFile(require.resolve('./rules/create_rules'));
-    loadTestFile(require.resolve('./rules/create_ml_rules_privileges'));
-    loadTestFile(require.resolve('./alerts/open_close_alerts'));
-    loadTestFile(require.resolve('./alerts/query_alerts'));
-    loadTestFile(require.resolve('./alerts/query_alerts_backword_compatibility'));
+  describe('Detection alerts Basic and Essentials API', function () {
```

Comment:
++ to @rylnd's suggestion. I find the naming of all our pipelines, steps, JUnit reports etc not hierarchical enough to be able to understand quickly (under 1 sec) what every name actually means, so I have to read each name carefully and "parse" it.

This could be improved in follow-up PRs, of course, not a blocker from my side.